languages/cpp/exercises/concept/numbers/.meta/example.h

```diff
@@ -0,0 +1,11 @@
+#ifndef NUMBERS_H
+#define NUMBERS_H
+
+#include <cstdint>
+
+namespace assembly_line {
+double production_rate_per_hour(const std::int32_t speed);
+std::int32_t working_items_per_minute(const std::int32_t speed);
+}  // namespace assembly_line
```

Comment:
Maybe add a little indentation here?

```suggestion
namespace assembly_line {
    double production_rate_per_hour(const std::int32_t speed);
    std::int32_t working_items_per_minute(const std::int32_t speed);
}
```

Reply:
The example implementation file uses the `const` modifier. As per [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#example-implementation-file), the example implementation should only use language features introduced by the exercise or its prerequisites (and their prerequisites, and so on). Is `const` introduced in one of the prerequisites?

Reply:
Besides, `const` value parameters should definitely not appear in declarations, as they don't do anything but confuse the reader.

Regarding indentation, we should probably all agree on a .clang-format document and stick to it; personally I never indent on namespaces

Reply:
I'm fine with whatever convention is used, as long as the same convention is used for all exercises.

Reply:
> Maybe add a little indentation here?

I have used the same '.clang-format' file as the one in the cpp repository with exercises to format this exercise.